TESTS/mbed_hal/qspi/main.cpp

```diff
@@ -321,10 +314,13 @@ void qspi_init_free_test(void)
     flash_init(qspi);
 
 #ifdef QSPI_TEST_LOG_FLASH_STATUS
-    printf("Status ");   log_register(STATUS_REG, QSPI_STATUS_REG_SIZE, qspi);
-    printf("Config 0 "); log_register(CONFIG_REG0, QSPI_CONFIG_REG_0_SIZE, qspi);
+    printf("Status\r\n");   log_register(STATUS_REG, QSPI_STATUS_REG_SIZE, qspi);
```

Comment:
You should either use config for t hat or configure terminal.

And I believe we should be using utest_printf rather. And not print this if all is OK?

Reply:
>You should either use config for t hat or configure terminal.

Don't understand this one

>And I believe we should be using utest_printf rather. And not print this if all is OK?

Sure I will change to utest_printf